### PR TITLE
Ghostdrone love part 1: Faces and emotes

### PR DESCRIPTION
**Pull request by NightmareChamillian**
[QoL] [wiki]
## About the PR 
This is an atomization of my two-part ghostdrone QoL project. This one lets ghostdrones answer yes-no questions, adds a special sprite for the C-RCD and adds eight new faces for drones to choose from. All facial sprites can be seen here:
image

Note that this doesn't add any sprites for the bedsheet variants of anything aside from the lopsy face, but adds the files so that they just show up as blank bedsheets instead of being invisible.

## Why's this needed? 
I think ghostdrones should be able to express individuality more easily. Also, being able to respond to someone's yes or no inquiries lets them communicate to a degree without giving them any ability to be pricks.

## Changelog

```changelog
(u)NightmareChamillian
(*)Adds seven new faces for ghostdrones to choose from! See minor changes for a list
(+)Faces are as follows: sleepy, exclamation mark, question mark, lopsy, kitty, and eye 
(+)Furthermore, drones can now nod (Ctrl+N) and shake (Ctrl+X) their heads to respond to basic yes-no questions
```

### CODE REVIEW

**Comment by NightmareChamillian** [comment]
Gallery: 
image
image
image

File: code/mob/living/silicon/ghostdrone.dm [hunk 1]
```diff
@@ -68,7 +68,7 @@
 		src.ears = src.radio
 
 		//Attach shit to tools
-		src.tools = list(
+		src.tools = list( // for part two: add 25 floor sheets, and a loofah(?)
```
Comment (pali6) [inline code/mob/living/silicon/ghostdrone.dm:71]:
```suggestion
		src.tools = list(
```

File: code/mob/living/silicon/ghostdrone.dm [hunk 2]
```diff
@@ -722,6 +723,16 @@
 					message = "<B>[src]</B> claps."
 					m_type = 2
 
+			if ("dance")
```
Comment (pali6) [inline code/mob/living/silicon/ghostdrone.dm:726]: Why not the `nod` emote instead? (Ctrl+N)

File: code/mob/living/silicon/ghostdrone.dm (hunk 1 above)
Comment (pali6) [reply code/mob/living/silicon/ghostdrone.dm:71]: Still not quite sure why this comment needs to be here.

**Review by frawhst — APPROVED** [review]

File: code/mob/living/silicon/ghostdrone.dm [hunk 3]
```diff
@@ -722,6 +723,16 @@
...
 					m_type = 2
 
+			if ("nod")  // we want it so ghostdrones can answer yes/no wuestions
+				if (!src.restrained())
+					message = "<B>[src]</B> nods its head yes."
+					m_type = 2
+
+			if ("snap")
+				if (!src.restrained())
+					message = "<B>[src]</B> shakes its head no."
```
Comment (ZeWaka) [inline code/mob/living/silicon/ghostdrone.dm:733]:
```suggestion
					message = "<B>[src]</B> nods its head."
					m_type = 2

			if ("snap")
				if (!src.restrained())
					message = "<B>[src]</B> shakes its head."
```

**Pull request merged**